src/lib/NotificationStoreAdapter.js

```diff
@@ -0,0 +1,37 @@
+export default class NotificationStoreAdapter {
```

Comment:
can be moved into `@DpJs/store/core/?`

Reply:
you're right! thanx!

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/71/commits/3797a37dbc5ca9cc122ca7168433323a5ddca775
New PR in demosplan-core: https://github.com/demos-europe/demosplan-core/pull/657